addons/l10n_uk/data/account.account.tag.csv

```diff
@@ -1,15 +1,10 @@
 id,name,applicability
-tag_st0,ST0,taxes
-tag_st1,ST1,taxes
-tag_st2,ST2,taxes
-tag_pt0,PT0,taxes
-tag_pt1,PT1,taxes
-tag_pt2,PT2,taxes
-tag_pt8m,PT8M,taxes
-tag_pt8r,PT8R,taxes
-tag_pt8,PT8,taxes
-tag_pt5,PT5,taxes
-tag_st4,ST4,taxes
-tag_pt7,PT7,taxes
-tag_st11,ST11,taxes
-tag_pt11,PT11,taxes
+tag_uk_01,Net VAT to be paid/reclaimed,taxes
+tag_uk_02,VAT on Sales and other outputs,taxes
```

Comment:
You should give better names to all your tags, as it was done in l10n_ch (here: https://github.com/odoo/odoo/blob/11.0/addons/l10n_ch/data/account_vat2011_data.xml ). Their ids should match the ones of the record lines they correspond to. 

For example, for tag_uk_02, you should name it vat_tag_0101, as it relates to line financial_report_line_uk_0101. Please do the same for all other tags.